### PR TITLE
session => analysis

### PR DESCRIPTION
**Pull request by dmfalke**
Rename "session" to "analysis".

### CODE REVIEW

**Review by jtlong3rd — CHANGES_REQUESTED** [review]
Almost good to merge! You'll want to address a few cases where `sessions` was replaced with `analysiss` instead of `analyses`.

File: src/lib/mapveu/Mocks.ts [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
 const localStore = localforage.createInstance({
-  name: 'mockSessionStore',
+  name: 'mockAnalysisStore',
 });
 
-export const mockSessionStore: SessionClient = {
-  async getSessions() {
-    const records: Session[] = [];
+export const mockAnalysisStore: AnalysisClient = {
+  async getAnalysiss() {
```
Comment (jtlong3rd) [inline src/lib/mapveu/Mocks.ts:13]: Here...

File: src/lib/workspace/Mocks.ts [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 const localStore = localforage.createInstance({
-  name: 'mockSessionStore',
+  name: 'mockAnalysisStore',
 });
 
-export const mockSessionStore: SessionClient = {
-  async getSessions() {
-    const records: Session[] = [];
+export const mockAnalysisStore: AnalysisClient = {
+  async getAnalysiss() {
```
Comment (jtlong3rd) [inline src/lib/workspace/Mocks.ts:11]: ... and here!

**Review by jtlong3rd — CHANGES_REQUESTED** [review]
Almost there! There are two lingering `analysiss`es.

**Comment by dmfalke** [comment]
Oops! Fixed.

**Review by jtlong3rd — APPROVED** [review]

**Pull request merged**